WRITING.md

```diff
@@ -154,6 +155,67 @@ This will render **Astro's latest version: v1.2.1**.
 
 The `<Version />` component is currently used in our Integrations pages as a way to keep each integration’s version up-to-date without having to merge changes to these pages directly. It's worth noting that this component is only updated at build-time, thus a package's version will not change until the site is rebuilt, be it manually or because a new PR was merged into main.
 
+### Tabs Components
+
+For pieces of content relevant to only a percentage of users (e.g. specific package manager or UI framework guidance), we have Tabs components to help you.
+
+For the two common cases: package managers and UI frameworks, the corresponding `<PackageManagerTabs>` and `<UIFrameworkTabs>` components already got you covered.
+
+It's worth mentioning that these components share state, so if you change the active tab of one `<PackageManagerTabs>` component, all of the other package manager Tabs components on the same page will also change.
+
+#### Examples
+
+All you have to do to use a Tabs component is to import it into the file and fill the available slots with content, for example:
+
+````jsx
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # create a new project with npm
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # create a new project with pnpm
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # create a new project with yarn
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+````
+You can take a look at how this example will be rendered on the [Automatic Astro Installation](https://docs.astro.build/en/install/auto/#1-run-the-setup-wizard) page.
+
+If necessary, you can also create your own Tabs component using the base `Tabs.tsx` component. To do this, create a new Astro component in the [`src/components/tabs`](https://github.com/withastro/docs/blob/main/src/components/tabs/) directory and fill `Tabs.tsx` with the corresponding tabs and panel slots, like the example below:
+
+```jsx
```

Comment:
```suggestion
```astro
```

Reply:
oh, GitHub supports Astro syntax highlighting in MD files? 🤔

Reply:
Ah good point, I don't know whether it does! For some reason, I had it in my head you'd SEE the language, and I didn't want it to seem confusing that you were documenting an Astro file but writing `jsx` - Probably jsx makes sense then!

Reply:
Confirmed, GitHub doesn't support Astro for syntax highlighting, unfortunately!